tests/profiles/universal_test.py

```diff
@@ -426,6 +437,25 @@ def deleteGlyphEncodings(font, cp):
                          FAIL, 'badA0',
                          'with bad glyph name for char 0x00A0 ...')
 
+  # restore the original font object in preparation for the next test-case:
+  ttFont = TTFont(TEST_FILE("mada/Mada-Regular.ttf"))
+
+  # See https://github.com/googlefonts/fontbakery/issues/2624
+  print("Test FAIL for naming 0x00A0 nbsp (it's not AGL-complient) ...")
+  editCmap(ttFont, 0x00A0, "nbsp")
+  assert_results_contain(check(ttFont),
+                         FAIL, 'badA0',
+                         'with bad glyph name for char 0x00A0 ...')
+
+  print("Test WARN for naming 0x00A0 nbspace ...")
+  editCmap(ttFont, 0x00A0, "nbspace")
+  assert_results_contain(check(ttFont),
+                         WARN, 'badA0')
+
+  print("Test PASS for naming 0x00A0 uni00A0 ...")
+  editCmap(ttFont, 0x00A0, "uni00A0")
+  assert_PASS(check(ttFont))
+
```

Comment:
Please remove the explicit `print` statements. The messages should be instead passed as a last argument to the assert_PASS function this way:

```python
  editCmap(ttFont, 0x00A0, "uni00A0")
  assert_PASS(check(ttFont),
              "for naming 0x00A0 uni00A0 ...")
```

Please make similar changes to the other ones as well